samples/sensor/die_temp_polling/boards/esp32s3_devkitm.overlay

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) 2023 Espressif Systems (Shanghai) Co., Ltd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Application overlay for creating temperature sensor device instance
```

Comment:
Actually, a nit (which I realize should be probably be fixed in other ESP32 overlays as well): 

```suggestion
 * Application overlay for enabling temperature sensor device instance
```